### ISSUE TITLE
Documentation: GRUB setting and customization

### ISSUE THREAD

**Issue by mudler** [rancher-sandbox/cOS-toolkit#237]
**Is your feature request related to a problem? Please describe.**
We miss docs about `/etc/cos/bootargs.cfg`, `/run/cos/cos-layout.env`

cos-layout.env:
(quoted from @davidcassany )
`This file only considers the three variable that are currently there. A volumes list, overlay values and debug flag for RW mounts which is currently unused. It does not create any partition or anything like that it just sets the mount points.
volumes are expressed as:
<id_type>=<id>:<mountpoint> , supported ID types are UUID and LABEL.
overlay can be configured with tmpfs:<size> or with <identifier_type>:<id> in case you want to use a persistent block device for the overlayfs.
Basically it maps the kernel params of the cos-immutable-rootfs dracut module defined in cOS-toolkit/packages/immutable-rootfs/30cos-immutable-rootfs/parse-cos-overlay.sh
`
bootargs.cfg:
`A list of commands that sets the kernel/initramfs parameters for the current image to boot`
